source/renderer/app/api/utils/patchAdaApi.js

```diff
@@ -185,4 +187,28 @@ export default (api: AdaApi) => {
   api.setNetworkBlockHeight = async (height: number) => {
     NETWORK_BLOCK_HEIGHT = height;
   };
+
+  api.setFakeNewsFeedJsonForTesting = (json: ?GetNewsResponse) => {
+    FAKE_NEWSFEED_JSON = json;
```

Comment:
shouldn't we change this variable name?

Reply:
@daniloprates you mean `json` one?

Reply:
@daniloprates I will change it to `fakeNewsfeedJson`